crate/src/lib.rs

```diff
@@ -63,40 +70,50 @@ pub fn bin_path() -> anyhow::Result<PathBuf> {
         return Ok(path);
     }
 
-    let mut buf = download_path();
+    let mut buf = download_path(version);
     buf.push("near-sandbox");
 
     Ok(buf)
 }
 
+pub fn normalize_name<S: AsRef<str>>(input: S) -> String {
+    // replace occurrences of '/' with '_'
+    let re = Regex::new(r"/").unwrap();
+    re.replace_all(input.as_ref(), "_").to_string()
+}
```

Comment:
1. Let's not make it `pub` as I don't think it is worth maintaining API compatibility in the future.
2. There is no reason to have `AsRef<str>` as all the calls for this function have `&str` as input
3. There is no need for Regex for a simple replacement (don't forget to remove it from the dependencies)

```suggestion
fn normalize_name(input: &str) -> String {
    input.as_ref().replace('/', "_")
}
```